tracer/test/test-applications/aspnet/Samples.Owin.Iis.WebApi2/Controllers/HomeController.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Web.Http;
+
+namespace Samples.Owin.WebApi2.Controllers
+{
+    [RoutePrefix("")]
+    public class HomeController : System.Web.Http.ApiController
+    {
+        [HttpGet]
+        [Route("alive-check")]
+        public string IsAlive()
+        {
+            return "Yes";
+        }
+
+        [HttpGet]
+        [Route("shutdown")]
+        public string Shutdown()
+        {
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+
+            foreach (var assembly in assemblies)
+            {
+                foreach (var type in assembly.DefinedTypes)
+                {
+                    if (type.Namespace == "Coverlet.Core.Instrumentation.Tracker")
```

Comment:
What exactly is the purpose of doing this?

Reply:
Honestly, I can't remember exactly 😅 It's a duplicate of the code we already have in the existing Owin tests - short answer is it's so that Coverlet doesn't bork. @kevingosse added it in [this PR](https://github.com/DataDog/dd-trace-dotnet/pull/1527) when he initially added Coverlet support 🙂

Reply:
Thanks :)